<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoveShape</class>
 <widget class="QDialog" name="MoveShape">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>131</width>
     <height>73</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter_4">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Shape ID: </string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="xLine"/>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Y: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="yLine"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MoveShape</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MoveShape</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
